decidim-proposals/app/controllers/decidim/proposals/admin/proposal_answers_controller.rb

```diff
@@ -35,6 +35,25 @@ def update
           end
         end
 
+        def update_multiple_answers
+          enforce_permission_to(:create, :proposal_answer)
+
+          if missing_cost_data?(proposals)
```

Comment:
```suggestion
          if answer_forms_invalid?(proposals)
```
I would make this more generic. For instance, you can define an incomplete template that would be invalid, regardless of the cost. This kind of template could not be used either